PhysicsTools/PatAlgos/python/triggerLayer1/triggerMatcherExamples_cfi.py

```diff
@@ -180,3 +180,19 @@
 , resolveAmbiguities    = cms.bool( True )        # only one match per trigger object
 , resolveByMatchQuality = cms.bool( True )        # take best match found per reco object: by DeltaR here (s. above)
 )
+
+triggerMatcherExamplesTask = cms.Task(
+    somePatMuonTriggerMatchHLTMu17,
```

Comment:
can this be refactored somewhat, e.g. triggerMatcherExamplesTask extended by one module right after each one is added?
Note that with ``` _exampleTriggerMatchers ``` we get the third place in this file to edit if some changes are needed